packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -116,20 +124,19 @@ const propTypes = {
    * The value must be in the `YYYY-MM-DD` format or the all-numeric date format based on the locale.
    */
   value: PropTypes.string,
-  /**
-   * @private
-   * Prop to show inline version of date picker component.
-   */
-  isInline: PropTypes.bool,
 };
 
 const defaultProps = {
+  ariaLabel: undefined,
```

Comment:
This is undefined by default. We don't need to add it to this list

Reply:
Removed them here: https://github.com/cerner/terra-framework/pull/1208/commits/80162e0e8172a99f65f5de3b9813ac2f3b8bede5